etl/scripts/faostat/archive_old_datasets.py

```diff
@@ -0,0 +1,267 @@
+"""Archive unused grapher FAOSTAT datasets and etl steps.
+
+TODO: Most of the logic in this script could (and eventually should) be integrated with VersionTracker.
+
+"""
+
+import argparse
+from typing import List
+
+import pandas as pd
+from MySQLdb.connections import Connection
+from structlog import get_logger
+
+from etl import db
+from etl.helpers import VersionTracker
+from etl.scripts.faostat.create_new_steps import find_latest_version_for_step
+from etl.scripts.faostat.shared import NAMESPACE
+
+log = get_logger()
+
+INTERACTIVE = True
+
+
+def list_active_db_datasets(db_conn: Connection) -> pd.DataFrame:
+    query = """
+    SELECT d.id, d.name, d.namespace, d.version, d.shortName
+    FROM datasets d
+    JOIN variables v
+    ON v.datasetId = d.id
+    JOIN chart_dimensions cd
+    ON v.id = cd.variableId
+    WHERE d.isArchived IS FALSE
+    ;
+    """
+    with db_conn.cursor() as cursor:
+        cursor.execute(query)
+        result = cursor.fetchall()
+    df = pd.DataFrame(result, columns=["dataset_id", "dataset_title", "namespace", "version", "short_name"])
```

Comment:
Maybe you have already considered this, but you can also use

```python
df = pd.read_sql(query, db.get_engine())
```

and then rename columns

```python
df = df.rename(columns={...})
```

Reply:
Thanks, I was looking for something like this, but didn't see the `get_engine` function. Anyway, not worth changing for this script. If we migrate to `VersionTracker` then I'll improve this.